cypress/support/config/services.js

```diff
@@ -27,9 +27,15 @@ module.exports = {
           Cypress.env('APP_ENV') === 'live'
             ? undefined
             : '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
-        smoke: true,
+        smoke: false,
```

Comment:
Not sure why this was set to true but for consistency setting it to false

Reply:
Is it not something to do with having one nominated service for that page type to be smoke tested in CI?

Reply:
Potentially, I don't know the requirements behind it. My understanding is that we are only smoke testing pages that are in production. Happy either way, just seemed inconsistent with the rest of the config

Reply:
Yeh I think it is true for the reason I mentioned. @benjaminhobbs can confirm